### ISSUE TITLE
[Accessibility] “Blocked by project” versions of SDK “Microsoft.NETCore.App” should be shown in grey instead of black

### ISSUE THREAD

**Issue by PatoBeltran** [NuGet/Home#5948]
**VSTS ISSUE:** 484554
**NuGet Version:** NuGet 4.4.0-Dev-4438
**VS Version:** D15PreRel 26823.3006
**OS Version:** Windows 10 RS2_Release 15063.2.170323-1837
 
## Repro Steps:
New a Visual C#->.NET Core->Console App (.NET Core) project.
Open PM UI of this project and go to “Installed” tab.
Select the SDK “Microsoft.NETCore.App” in the left panel.
Click the “Version” drop-down list and check the versions.

## Expected Result:
These “blocked by project” versions should be shown in grey (same with the color of “Uninstall” and “Update” buttons).
 
## Actual Result:
These “blocked by project” versions were shown in black as below screenshot.

![attachfilehandler](https://user-images.githubusercontent.com/2132567/30941215-3009c9f4-a399-11e7-98fd-e49ac1b21968.png)
